panther_battery/include/panther_battery/roboteq_republisher_node.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef PANTHER_BATTERY_ROBOTEQ_REPUBLISHER_NODE_HPP_
+#define PANTHER_BATTERY_ROBOTEQ_REPUBLISHER_NODE_HPP_
+
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <sensor_msgs/msg/battery_state.hpp>
+
+#include <panther_msgs/msg/driver_state.hpp>
+
+#include <panther_utils/moving_average.hpp>
+
+namespace panther_battery
+{
+using BatteryStateMsg = sensor_msgs::msg::BatteryState;
+using DriverStateMsg = panther_msgs::msg::DriverState;
+
+class RoboteqRepublisherNode : public rclcpp::Node
+{
+public:
+  RoboteqRepublisherNode();
+
+private:
+  static constexpr float bat_capacity_ = 20.0f;
+  static constexpr float bat_designed_capacity_ = 20.0f;
+  static constexpr float V_bat_fatal_min_ = 27.0f;
+  static constexpr float V_bat_fatal_max_ = 43.0f;
+  static constexpr float V_bat_full_ = 41.4f;
+  static constexpr float V_bat_min_ = 32.0f;
```

Comment:
```suggestion
  static constexpr float V_bat_min_ = 32.0f;
  static constexpr std::string_view location_ = "user_compartment";
```
This might be a way to add const version of location name